modules/solid_mechanics/test/tests/modal_analysis/tests

```diff
@@ -0,0 +1,11 @@
+[Tests]
+  issues = '#27295'
+  [modal]
+    type = Exodiff
+    input = 'modal.i'
+    exodiff = 'modal_out.e'
+    exodiff_opts = '-eigen_relative'
```

Comment:
This is for eigenvectors since -1 * vector is still a correct eigenvectors. Had problems in parallel.

Reply:
Are the eigenvectors scaled?

Reply:
What do you mean scaled? I meant the eigenvectors make point in the opposite direction depending on parallel vs serial.

Reply:
The overall shape is the same but it has an arbitrary scale, like multiplying all of the displacements of these eigenvectors by 5 is still the same eigenvector.  Sometimes the eigenvectors are normalized to have a maximum value of 1, just wondering if they do this here.